scripts/bert/dataset.py

```diff
@@ -67,7 +440,8 @@ def _truncate_seq_pair(tokens_a, tokens_b, max_length):
         else:
             tokens_b.pop()
 
-class BERTTransform(object):
+
+class BERTTransform:
```

Comment:
please keep inheritance from object for python2/3 compatibility

Reply:
Updated.